docs/images/Logo_no_background.svg

```diff
@@ -0,0 +1,3 @@
Rbo8ShYG8MXtJAQIIIIBAxQK/HuwLYkHuePaXtfv2P7C//D1/cn9JBv6y7s6VIrpZor2isGJOtIKd7JK7dbggBBBAAIFfCZTf7meaHZOgvxSxdXv6h3ekppNc4C9b11kOQXQWMdai5XlqYFwPAggggAACpxIIWqtlIeQq+njj+PEP7vv0U4dS0Uou8Jd3L7Jmfvw4i96ncotwHQgggAACpyNQBmu9Nnv2YP9QUhmb1MWUoGXgN/Lj03rHotO5cahFAAEEEGg/gQ4Cf+ymEfhjG1GBAAIIIJC2AIHv6A+B70CiBAEEEEAgaQEC39EeAt+BRAkCCCCAQNICBL6jPQS+A4kSBBBAAIGkBQh8R3sIfAcSJQgggAACSQsQ+I72EPgOJEoQQAABBJIWIPAd7SHwHUiUIIAAAggkLUDgO9pD4DuQKEEAAQQQSFqAwHe0h8B3IFGCAAIIIJC0AIHvaA+B70CiBAEEEEAgaQEC39EeAt+BRAkCCCCAQNICBL6jPQS+A4kSBBBAAIGkBQh8R3sIfAcSJQgggAACSQsQ+I72EPgOJEoQQAABBJIWIPAd7SHwHUiUIIAAAggkLUDgO9pD4DuQKEEAAQQQSFqAwHe0h8B3IFGCAAIIIJC0AIHvaA+B70CiBAEEEEAgaQEC39EeAt+BRAkCCCCAQNICBL6jPQS+A4kSBBBAAIGkBQh8R3sIfAcSJQgggAACSQsQ+I72EPgOJEoQQAABBJIWIPAd7SHwHUiUIIAAAggkLUDgO9pD4DuQKEEAAQQQSFqAwHe0h8B3IFGCAAIIIJC0AIHvaA+B70CiBAEEEEAgaQEC39EeAt+BRAkCCCCAQNICBL6jPQS+A4kSBBBAAIGkBQh8R3sIfAcSJQgggAACSQsQ+I72EPgOJEoQQAABBJIWIPAd7SHwHUiUIIAAAggkLUDgO9pD4DuQKEEAAQQQSFqAwHe0h8B3IFGCAAIIIJC0AIHvaA+B70CiBAEEEEAgaQEC39EeAt+BRAkCCCCAQNICBL6jPQS+A4kSBBBAAIGkBQh8R3sIfAcSJQgggAACSQsQ+I72EPgOJEoQQAABBJIWIPAd7SHwHUiUIIAAAggkLUDgO9pD4DuQKEEAAQQQSFqAwHe0h8B3IFGCAAIIIJC0AIHvaA+B70CiBAEEEEAgaQEC39EeAt+BRAkCCCCAQNICBL6jPQS+A4kSBBBAAIGkBQh8R3sIfAcSJQgggAACSQsQ+I72EPgOJEoQQAABBJIWIPAd7SHwHUiUIIAAAggkLUDgO9pD4DuQKEEAAQQQSFqAwHe0h8B3IFGCAAIIIJC0AIHvaA+B70CiBAEEEEAgaQEC39EeAt+BRAkCCCCAQNICBL6jPQS+A4kSBBBAAIGkBQh8R3sIfAcSJQgggAACSQsQ+I72EPgOJEoQQAABBJIWIPAd7SHwHUiUIIAAAggkLUDgO9pD4DuQKEEAAQQQSFqAwHe0h8B3IFGCAAIIIJC0AIHvaA+B70CiBAEEEEAgaQEC39EeAt+BRAkCCCCAQNICBL6jPQS+A4kSBBBAAIGkBQh8R3sIfAcSJQgggAACSQsQ+I72EPgOJEoQQAABBJIWIPAd7SHwHUiUIIAAAggkLUDgO9pD4DuQKEEAAQQQSFqAwHe0h8B3IFGCAAIIIJC0AIHvaA+B70CiBAEEEEAgaQEC39EeAt+BRAkCCCCAQNICBL6jPQS+A4kSBBBAAIGkBQh8R3sIfAcSJQgggAACSQsQ+I72EPgOJEoQQAABBJIWIPAd7SHwHUiUIIAAAggkLUDgO9pD4DuQKEEAAQQQSFqAwHe0h8B3IFGCAAIIIJC0AIHvaA+B70CiBAEEEEAgaQEC39EeAt+BRAkCCCCAQNICBL6jPQS+A4kSBBBAAIGkBQh8R3sIfAcSJQgggAACSQsQ+I72EPgOJEoQQAABBJIWIPAd7SHwHUiUIIAAAggkLUDgO9pD4DuQKEEAAQQQSFqAwHe0h8B3IFGCAAIIIJC0AIHvaA+B70CiBAEEEEAgaQEC39EeAt+BRAkCCCCAQNICBL6jPQS+A4kSBBBAAIGkBQh8R3sIfAcSJQgggAACSQsQ+I72EPgOJEoQQAABBJIWIPAd7SHwHUiUIIAAAggkLUDgO9pD4DuQKEEAAQQQSFqAwHe0h8B3IFGCAAIIIJC0AIHvaA+B70CiBAEEEEAgaQEC39EeAt+BRAkCCCCAQNICBL6jPQS+A4kSBBBAAIGkBQh8R3sIfAcSJQgggAACSQsQ+I72EPgOJEoQQAABBJIWIPAd7SHwHUiUIIAAAggkLUDgO9pD4DuQKEEAAQQQSFqAwHe0h8B3IFGCAAIIIJC0AIHvaA+B70CiBAEEEEAgaQEC39EeAt+BRAkCCCCAQNICBL6jPQS+A4kSBBBAAIGkBQh8R3sIfAcSJQgggAACSQsQ+I72EPgOJEoQQAABBJIWIPAd7SHwHUiUIIAAAggkLUDgO9pD4DuQKEEAAQQQSFqAwHe0h8B3IFGCAAIIIJC0AIHvaA+B70CiBAEEEEAgaQEC39EeAt+BRAkCCCCAQNICBL6jPQS+A4kSBBBAAIGkBQh8R3sIfAcSJQgggAACSQsQ+I72EPgOJEoQQAABBJIWIPAd7SHwHUiUIIAAAggkLUDgO9pD4DuQKEEAAQQQSFqAwHe0h8B3IFGCAAIIIJC0AIHvaA+B70CiBAEEEEAgaQEC39EeAt+BRAkCCCCAQNICBL6jPQS+A4kSBBBAAIGkBQh8R3sIfAcSJQgggAACSQsQ+I72EPgOJEoQQAABBJIWIPAd7SHwHUiUIIAAAggkLUDgO9pD4DuQKEEAAQQQSFqAwHe0h8B3IFGCAAIIIJC0AIHvaA+B70CiBAEEEEAgaQEC39EeAt+BRAkCCCCAQNICBL6jPQS+A4kSBBBAAIGkBQh8R3vKwHeUUYIAAggggEDSAoP9Q5rSBf4fd2sL5QvJHdYAAAAASUVORK5CYII="/>
```

Comment:
I am kind of lost here, I thought it was meant to be the full SVG of the logo? Looks like it's still a PNG but embedded in a SVG

Reply:
Ah, I should have checked the svg with the inspector first. Will make the necessary changes. That explains the poor image quality as well.